namui-cli/src/build/build.rs

```diff
@@ -59,7 +54,7 @@ fn print_build_result(error_messages: &Vec<ErrorMessage>, cli_error_messages: &V
     println!("Errors {}", error_messages.len() + cli_error_messages.len());
     for error_message in error_messages {
         println!(
-            "{}\n\t--> {}:{}:{}\n",
+            "{}\n\t\x1b[34m--> {}:{}:{}\x1b[0m\n",
```

Comment:
`\x1b[34m` is command that setting color.
`\x1b[0m` is command that clearing color.